README.md

```diff
@@ -32,7 +32,7 @@
 - 'git remote -v': look at all the remotes you have
 - 'git push <WHERE> <WHAT>': pushes the <WHAT> branch to the <WHERE>
     - 'git push origin main
-- 'git pull <WHERE> <WHAT>': pulls the <WHAT> branch in <WHERE> to local computer
+- 'git pull <WHERE> <WHAT>': pulls the <WHAT> branch in <WHERE> to orig local computer
```

Comment:
Interesting that this line is different than the class. must have something to do with the fact that I pushed it before?  Thet does not make sense since I deleted the last push request from the desktop?